<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MouseMover</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Detenido</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Offset:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Interval (sec)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="intervalEdit">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="offset">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startBtn">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>384</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>startBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>toggleMovement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intervalEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshInterval(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setMovementArea()</slot>
  <slot>toggleMovement()</slot>
  <slot>refreshInterval(int)</slot>
  <slot>saveSettings()</slot>
 </slots>
</ui>
